core/src/main/scala/org/typelevel/paiges/Doc.scala

```diff
@@ -523,6 +523,7 @@ sealed abstract class Doc extends Product with Serializable {
 }
 
 object Doc {
+  val DEFAULT_TRIM = false
```

Comment:
I personally prefer `DefaultTrim` for constants like this. I don't know that we have an established rule on this project, yet. Seems more common in the Scala community.